test.sh

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
+set -eux
 
 FAILURE=0
 
+# Array of PostgreSQL versions for testing
+PG_VERSIONS=(9.5 9.6 10 11 12 13 14 15 16)
+
 # Stop any existing containers from previous test runs
 test_down() {
-    docker-compose -f test/docker-compose-pgauto.yml down
+    docker compose -f test/docker-compose-pgauto.yml down
```

Comment:
Yeah, good point. :smile:

Reply:
My local Docker installation complained that it no longer knows about `docker-compose`. :D

Reply:
Yeah.  Have some a mix of both new and old compose across servers still, though it's mostly the new compose v2 these days. :smile: